mlrun/common/schemas/model_monitoring/constants.py

```diff
@@ -234,6 +234,7 @@ class MonitoringTSDBTables(MonitoringStrEnum):
     APP_RESULTS = "app-results"
     METRICS = "metrics"
     EVENTS = "events"
+    INVOCATIONS = "invocations"
```

Comment:
It doesn't belong to `MonitoringTSDBTables`.

Add it to a new place, including the `"mlrun-infra" app name.

Reply:
I created a constant for `"mlrun-infra"`, but I don't know where you'd like to move `INVOCATIONS` (it's a TSDB table for monitoring, so I'm not sure why it wouldn't belong in `MonitoringTSDBTables`).

Reply:
> I created a constant for `"mlrun-infra"`,

Ok, please use it in other places (see above)

> but I don't know where you'd like to move `INVOCATIONS` (it's a TSDB table for monitoring, so I'm not sure why it wouldn't belong in `MonitoringTSDBTables`).

The table you use is:
```py
f"pipelines/{project}/model-endpoints/predictions"
```
"invocations" is just the metric name.